test/tests/mesh_modifiers/sidesets_between_subdomains/between_vector_blocks.i

```diff
@@ -0,0 +1,27 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 2
+  ny = 2
+  nz = 0
+  zmin = 0
+  zmax = 0
+  elem_type = QUAD4
+[]
+
+[MeshModifiers]
+  [./subdomain_id]
+    type = AssignElementSubdomainID
+    subdomain_ids = '0 0
+                     1 2'
+  [../]
+  [./extrude]
```

Comment:
Yeah, I will change "extrude" to "interface" it was from the original test "between_in.i"

K. Then I guess for the input file, I name it sidesets_between_vectors_of_subdomains.i.